src/main/java/org/spongepowered/api/placeholder/PlaceholderParsers.java

```diff
@@ -22,47 +22,32 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.event.entity.living;
+package org.spongepowered.api.placeholder;
 
-import org.spongepowered.api.entity.living.Humanoid;
-import org.spongepowered.api.event.Cancellable;
-import org.spongepowered.api.event.Event;
-import org.spongepowered.api.util.annotation.eventgen.GenerateFactoryMethod;
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
 
 /**
- * Called when a human's level is changed.
+ * Contains Sponge provided {@link PlaceholderParser}s.
  */
-@GenerateFactoryMethod
-public interface ChangeLevelEvent extends Event, Cancellable {
+public class PlaceholderParsers {
 
-    /**
-     * Gets the {@link Humanoid}.
-     *
-     * @return The humanoid
-     */
-    Humanoid getHumanoid();
+    // SORTFIELDS: ON
 
     /**
-     * Gets the original level of the human.
-     *
-     * @return The original level of the human
+     * A parser that returns the associated source's current world, if
+     * applicable, else the default world.
      */
-    int getOriginalLevel();
+    static PlaceholderParser CURRENT_WORLD = DummyObjectProvider.createFor(PlaceholderParser.class, "CURRENT_WORLD");
```

Comment:
Should turn into Suppliers, `DummyObjectProvider` is gonna be nuked.

Reply:
Done